iamy/aws.go

```diff
@@ -176,6 +190,27 @@ func (a *AwsFetcher) marshalRoleDescriptionAsync(roleName string, target *string
 	}()
 }
 
+func (a *AwsFetcher) populateInstanceProfileData(resp *iam.ListInstanceProfilesOutput) error {
+	for _, profileResp := range resp.InstanceProfiles {
+		if cfnResourceRegexp.MatchString(*profileResp.InstanceProfileName) {
+			log.Printf("Skipping CloudFormation generated instance profile %s", *profileResp.InstanceProfileName)
+			continue
+		}
+		profile := InstanceProfile{iamService: iamService{
+			Name: *profileResp.InstanceProfileName,
+			Path: *profileResp.Path,
+		}}
+		if len(profileResp.Roles) > 0 {
+			for _, roleResp := range  profileResp.Roles {
+				role := *(roleResp.RoleName)
+				profile.Roles = append (profile.Roles, role)
```

Comment:
we've got an extra bit of whitespace after the `append` keyword here.

Reply:
I like whitespace.
But fixed in 043328d